src/nbl/video/CVulkanBuffer.cpp

```diff
@@ -8,7 +8,7 @@ namespace nbl::video
 CVulkanBuffer::~CVulkanBuffer()
 {
 	preDestroyStep();
-	if (m_cachedCreationParams.skipHandleDestroy)
+	if (!m_cachedCreationParams.skipHandleDestroy)
```

Comment:
what a typo! did I mess up the same thing anywhere else?

Reply:
Other dtors were fine